docs/aws-china.md

```diff
@@ -254,10 +254,25 @@ done
 
 ## Upload assets
 
+## Get default S3 multipart_threshold
+
+AWS_S3_DEFAULT_MULTIPART_THRESHOLD=$(aws configure get default.s3.multipart_threshold)
+
+if [ ! -n "$AWS_S3_DEFAULT_MULTIPART_THRESHOLD" ]; then
+  AWS_S3_DEFAULT_MULTIPART_THRESHOLD=8MB
+fi
+
+## Set multipart_threshold to 1024MB to prevent Etag not returns MD5 when upload multipart
+
+aws configure set default.s3.multipart_threshold 1024MB
```

Comment:
I feel there must be a way to specify this only for this instance, but I looked and I couldn't find it either!